rpc/wrtc_signaling_server.go

```diff
@@ -331,6 +349,28 @@ func (srv *WebRTCSignalingServer) Answer(server webrtcpb.SignalingService_Answer
 	}
 	defer srv.clearAdditionalICEServers(hosts)
 
+	// If heartbeats allowed (indicated by answerer), start goroutine to send
+	// heartbeats.
+	if HeartbeatsAllowedFromCtx(ctx) {
+		utils.PanicCapturingGo(func() {
+			for {
+				select {
+				case <-time.After(heartbeatInterval):
+					if err := server.Send(&webrtcpb.AnswerRequest{
+						Stage: &webrtcpb.AnswerRequest_Heartbeat{},
+					}); err != nil {
+						srv.logger.Debugw(
```

Comment:
Similar remark to the other comment re: side-effects. I buy these changes fix the long delays before robots can be connected to again. But because we just ignore the errors, it's going to be hard for a reader to pick up our expectation of what's changing under the hood. I know our information here is limited, but sometimes it's important to communicate what we don't know just as much as it is communicating what we do know.